moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -1357,21 +1357,21 @@ LinkModel* RobotModel::getLinkModel(const std::string& name, bool* has_link)
   {
     *has_link = false;  // Report failure via argument
   }
-  else
-  {  // Otherwise print error
-    RCLCPP_ERROR(getLogger(), "Link '%s' not found in model '%s'", name.c_str(), model_name_.c_str());
-  }
```

Comment:
The error message was printed only if the caller didn't provide the has_link argument to handle error checking itself. Probably, now you will get duplicates of the error message.

Reply:
I deprecated the argument, this pattern is just ugly and prone to errors